pcweb/pages/docs/advanced_guide/wrapping_react.py

```diff
@@ -43,7 +43,7 @@ class ColorPicker(pc.Component):
     color: pc.Var[str]
 
     @classmethod
-    def get_controlled_triggers(cls) -> set[str]:
+    def get_controlled_triggers(cls) -> Set[str]:
```

Comment:
This is valid from Python 3.9+ so we don't have to change it

Reply:
Fair enough. I thought it was a typo because pynecone repo uses old syntax.